### ISSUE TITLE
Docker Hub image is update but with no content changes

### ISSUE THREAD

**Issue by fank** [mbentley/docker-teamspeak#11]
I'm running a auto update for docker images, and it will restart container if there are updated on the docker registry.
The docker image on docker hub, should only be update if there is a new version of teamspeak available, or something changed in this repository.

**Comment by fank** [comment]
And we also should try to add a tag of the teamspeak version as well.

**Comment by mbentley** [comment]
Yeah, right now the image would be updated if there is a post commit hook to master or if there has been a push to the upstream base image repo; the latter is probably causing all of the new image builds since auto builds on hub doesn't cache.  Unfortunately that means that we do more rebuilds than we actually need since it will trigger a build for the different tags but it ensures we get the latest base image.  It would be great to do builds for specific versions as they are available; would probably need to modify the image to install a specific version and then start using releases.  We could then have Docker Hub automatically build  based off of the release.  Thoughts?

**Comment by fank** [comment]
In example the teamspeak version is "3.0.13.5", we have 5 tags on it:
- latest
- 3
- 3.0
- 3.0.13
- 3.0.13.5

"latest" ill always be updated based on the base, same as currently.
And the other 4 tags will only be created if not exists, like if there is an update to "3.0.13.6" following tags will be created:
- 3
- 3.0
- 3.0.13
- 3.0.13.6

So on a base image update it detects that the "3.0.13.6" tag exists and wont push a new image with this tag.

So we have:
- a up2date image to ensure it meets the security standards
- a stable image which will be stable and never changed to ensure it has a long update, and be update with current releases.

We also should only update current images, so now "3.0.13.6" is the current version, we wont change anything in the "3.0.13.5" image.
